debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnector.java

```diff
@@ -102,10 +101,11 @@ public Config validate(Map<String, String> connectorConfigs) {
             try (PostgresConnection connection = new PostgresConnection(config.jdbcConfig())) {
                 connection.execute("SELECT version()");
                 logger.info("Successfully tested connection for {} with user '{}'", connection.connectionString(),
-                    connection.username());
-            } catch (Throwable e) {
+                        connection.username());
+            }
+            catch (Throwable e) {
                 logger.info("Failed testing connection for {} with user '{}'", config.jdbcConfig(),
-                    userValue);
+                        userValue);
```

Comment:
Are these changes necessary? Appear to be whitespace-only.

Reply:
Yeah... `mvn clean test` auto-formatted these files and I just left them. Probably debezium enforces a slightly different code style.